pkg/scheduler/framework/plugins/dynamicresources/dynamicresources_test.go

```diff
@@ -1114,13 +1118,18 @@ type testContext struct {
 
 func (tc *testContext) verify(t *testing.T, expected result, initialObjects []metav1.Object, result interface{}, status *framework.Status) {
 	t.Helper()
-	if expected.status == nil {
-		assert.Nil(t, status)
+	if expected.status == nil && status != nil {
+		t.Errorf("Expected nil status but got: %+v", status)
```

Comment:
Maybe keep the previous logic:
```suggestion
	if expected.status == nil {
	    if status != nil {
	        t.Errorf("Expected nil status but got: %+v", status)
	    }
```

Reply:
I got a comment to simplify it 😅